plugins/eslint-plugin-liferay/lib/rules/sort-import-destructures.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * © 2017 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+const DESCRIPTION = 'destructured names in imports must be sorted';
+
+module.exports = {
+	create(context) {
+		return {
+			ImportDeclaration(node) {
+				const specifiers = node.specifiers.filter(specifier => {
+					// Just `ImportSpecifier` (ignore `ImportDefaultSpecifier`).
+					return specifier.type === 'ImportSpecifier';
+				});
+
+				if (specifiers.length > 1) {
+					const source = context.getSourceCode();
+
+					if (
+						source.commentsExistBetween(
+							source.getTokenBefore(specifiers[0]),
+							node.source
+						)
+					) {
+						// Don't touch if any of the specifiers have
+						// comments.
```

Comment:
Having comments intermingled among import specifiers should be exceedingly rare (I've never seen an example in our codebases) so I'm avoiding the complexity required to handle comments. If we ever have a legitimate example of such a comment, we can do a follow-up.